excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/validation/SpreadSheetValidationMappingEnum.java

```diff
@@ -15,7 +15,6 @@
 
```

Comment:
No changes to unit tests, already covered by current set of tests in SpreadsheetValidatorTest